app/models/book.rb

```diff
@@ -1,4 +1,4 @@
 class Book < ApplicationRecord
-  has_many :copies, dependent: :destroy
+  has_many :copies, foreign_key: :book_id, dependent: :destroy
```

Comment:
Ah, sorry! I meant adding a foreign key to the database, so that the database enforces the Book-Copy relationship (in theory Rails should enforce it for us, but doing it at the DB level adds an extra level of safety): https://guides.rubyonrails.org/active_record_migrations.html#foreign-keys